### PR TITLE
[hailtop.utils]added new test for unzip

### PR DESCRIPTION
**Pull request by CDiaz96**

### CODE REVIEW

File: hail/python/test/hailtop/utils/test_utils.py [hunk 1]
```diff
@@ -181,3 +181,13 @@ def test_grouped_size_3_groups_7_elements():
     actual = list(grouped(3,['abc', 'def', 'ghi', 'jkl', 'mno', 'pqr', 'stu']))
     expected = [['abc', 'def', 'ghi'], ['jkl', 'mno', 'pqr'], ['stu']]
     assert actual == expected
+
+
+def test_unzip():
+    assert unzip([]) == ([], [])
+    assert unzip([(0, 'a')]) == ([0], ['a'])
+    assert unzip([(123, 'abc')]) == ([123], ['abc'])
+    assert unzip([(123, 456), ('abc', 'def')]) == ([123, 'abc'], [456, 'def'])
```
[the remark this answers is not in the document]
Comment (CDiaz96) [reply hail/python/test/hailtop/utils/test_utils.py:191]: Added empty string test

File: hail/python/test/hailtop/utils/test_utils.py [hunk 2]
```diff
@@ -181,3 +181,13 @@ def test_grouped_size_3_groups_7_elements():
...
+    assert unzip([(123, 'abc'), (456, 'def'), (789, 'ghi')]) == ([123, 456, 789], ['abc', 'def', 'ghi'])
+    
+
```
[the remark this answers is not in the document]
Comment (CDiaz96) [reply hail/python/test/hailtop/utils/test_utils.py]: removed extra lines

File: hail/python/test/hailtop/utils/test_utils.py [hunk 3]
```diff
@@ -186,8 +186,7 @@ def test_grouped_size_3_groups_7_elements():
 def test_unzip():
     assert unzip([]) == ([], [])
     assert unzip([(0, 'a')]) == ([0], ['a'])
+    assert unzip([(123,'')]) == ([123], [''])
     assert unzip([(123, 'abc')]) == ([123], ['abc'])
     assert unzip([(123, 456), ('abc', 'def')]) == ([123, 'abc'], [456, 'def'])
-    assert unzip([(123, 'abc'), (456, 'def'), (789, 'ghi')]) == ([123, 456, 789], ['abc', 'def', 'ghi'])
-    
-    
+    assert unzip([(123, 'abc'), (456, 'def'), (789, 'ghi')]) == ([123, 456, 789], ['abc', 'def', 'ghi'])
```
Comment (jigold) [inline hail/python/test/hailtop/utils/test_utils.py:192]: Do you see the red symbol at the end of the line above. That means you need a new line at the end of the file. You had two before. Can you fix this by adding a new line?

**Review by jigold — CHANGES_REQUESTED** [review]

File: hail/python/test/hailtop/utils/test_utils.py (hunk 3 above)
Comment (CDiaz96) [reply hail/python/test/hailtop/utils/test_utils.py:192]: Oh Okay. So, I need to keep at least 1 empty line at the end of the file.

File: hail/python/test/hailtop/utils/test_utils.py [hunk 4]
```diff
@@ -181,3 +181,13 @@ def test_grouped_size_3_groups_7_elements():
...
+
+
+def test_unzip():
+    assert unzip([]) == ([], [])
+    assert unzip([(0, 'a')]) == ([0], ['a'])
+    assert unzip([(123,'')]) == ([123], [''])
+    assert unzip([(123, 'abc')]) == ([123], ['abc'])
+    assert unzip([(123, 456), ('abc', 'def')]) == ([123, 'abc'], [456, 'def'])
+    assert unzip([(123, 'abc'), (456, 'def'), (789, 'ghi')]) == ([123, 456, 789], ['abc', 'def', 'ghi'])
+
```
Comment (danking) [inline hail/python/test/hailtop/utils/test_utils.py:193]: I think you might have four blank spaces on this line. Can you remove those? Thanks!

**Review by danking — CHANGES_REQUESTED** [review]

File: hail/python/test/hailtop/utils/test_utils.py (hunk 4 above)
Comment (danking) [reply hail/python/test/hailtop/utils/test_utils.py:193]: To be clear, one empty line is great, but it shouldn't have any characters on it.

Comment (CDiaz96) [reply hail/python/test/hailtop/utils/test_utils.py]: Okay. I see, it had a space in the line

File: hail/python/test/hailtop/utils/test_utils.py [hunk 5]
```diff
@@ -181,3 +181,12 @@ def test_grouped_size_3_groups_7_elements():
     actual = list(grouped(3,['abc', 'def', 'ghi', 'jkl', 'mno', 'pqr', 'stu']))
     expected = [['abc', 'def', 'ghi'], ['jkl', 'mno', 'pqr'], ['stu']]
     assert actual == expected
+
+
+def test_unzip():
+    assert unzip([]) == ([], [])
+    assert unzip([(0, 'a')]) == ([0], ['a'])
+    assert unzip([(123,'')]) == ([123], [''])
```
Comment (jigold) [inline hail/python/test/hailtop/utils/test_utils.py:189]: Can you put a space after the comma? `(123, '')`

**Review by jigold — CHANGES_REQUESTED** [review]

**Review by jigold — APPROVED** [review]

**Pull request merged**